src/Query/Builder.php

```diff
@@ -695,7 +704,9 @@ public function delete($id = null)
         }
 
         $wheres = $this->compileWheres();
-        $result = $this->collection->DeleteMany($wheres);
+        $options = $this->inheritConnectionOptions();
+
+        $result = $this->collection->DeleteMany($wheres, $options);
```

Comment:
I realize method names are case-insensitive but we should use the canonical case:

```suggestion
        $result = $this->collection->deleteMany($wheres, $options);
```